vault.yaml

```
@@ -1,9 +1,8 @@
 package:
   name: vault
-  version: 1.12.3
-  epoch: 1
-  description: Tool for encryption as a service, secrets and privileged access management
-    - !riscv64
+  version: 1.12.4
+  epoch: 0
+  description: Tool for encryption as a service, secrets and privileged access management - !riscv64
```

```suggestion
  description: Tool for encryption as a service, secrets and privileged access management
```

I think I did this on accident. 😵‍💫

---

Thanks for catching it!

---

@imjasonh I applied your suggestion which dismissed your review..... 😮‍💨